metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -327,52 +531,38 @@ def _get_bigquery_log_entries_via_gcp_logging(
         filter = self._generate_filter(BQ_AUDIT_V2)
         logger.debug(filter)
 
-        try:
```

Comment:
This is covered by the try/except in `_get_usage_events` and also makes this consistent with `_get_exported_bigquery_audit_metadata` which doesn't have a try/except